pkg/trace/writer/trace.go

```diff
@@ -58,9 +59,8 @@ type TraceWriter struct {
 	wg       sync.WaitGroup // waits for gzippers
 	tick     time.Duration  // flush frequency
 
-	traces       []*pb.APITrace // traces buffered
-	events       []*pb.Span     // events buffered
-	bufferedSize int            // estimated buffer size
+	tracerPayloads []*pb.TracerPayload // tracer payloads buffered
```

Comment:
```suggestion
	payloads []*pb.TracerPayload // tracer payloads buffered
```
Can we try to shorten things? Would this name I am proposing clash with anything and create confusion?

Reply:
I would like to keep it as `tracerPayloads`. Otherwise, it will be a bit confusing as there is another payload called Agent Payload.

Reply:
That's perfectly fine by me.